python_modules/libraries/dagster-fivetran/dagster_fivetran_tests/test_load_from_instance.py

```diff
@@ -307,3 +308,110 @@ def downstream_asset(xyz):
 
         # Validate IO manager is called to retrieve the xyz asset which our downstream asset depends on
         assert load_calls == [xyz_asset_key]
+
+
+@responses.activate
+def test_load_from_instance_with_translator() -> None:
+    with environ({"FIVETRAN_API_KEY": "some_key", "FIVETRAN_API_SECRET": "some_secret"}):
+        load_calls = []
+
+        @io_manager
+        def test_io_manager(_context) -> IOManager:
+            class TestIOManager(IOManager):
+                def handle_output(self, context: OutputContext, obj) -> None:
+                    return
+
+                def load_input(self, context: InputContext) -> Any:
+                    load_calls.append(context.asset_key)
+                    return None
+
+            return TestIOManager()
+
+        ft_resource = FivetranResource(
+            api_key=EnvVar("FIVETRAN_API_KEY"), api_secret=EnvVar("FIVETRAN_API_SECRET")
+        )
+
+        b64_encoded_auth_str = base64.b64encode(b"some_key:some_secret").decode("utf-8")
+        expected_auth_header = {"Authorization": f"Basic {b64_encoded_auth_str}"}
+
+        responses.add(
```

Comment:
since these are shared with other test, would be good to pull into shared helper method